gadget/mountedfilesystem.go

```diff
@@ -601,8 +615,7 @@ func (f *mountedFilesystemUpdater) Backup() error {
 		return fmt.Errorf("cannot create backup directory: %v", err)
 	}
 
-	preserveInDst, err := mapPreserve(f.mountPoint,
-		append(f.ps.Update.Preserve, f.managedBootAssets...))
+	preserveInDst, err := mapPreserve(f.mountPoint, f.ps.Update.Preserve)
```

Comment:
should this change be done here or in the follow up changes?  we haven't changed boot in this PR

Reply:
Heh, missed that spot, let me fix that.